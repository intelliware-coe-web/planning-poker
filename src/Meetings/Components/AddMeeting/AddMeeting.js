import React from 'react';
import {Link} from 'react-router-dom';

export default function AddMeeting() {
    
    return (
        <div className="uk-container uk-text-center@m uk-margin-top uk-margin-bottom">
            <h1 className="uk-heading-small uk-heading-divider">Add Meeting</h1>
            <a href="#/meetings">
                <span uk-icon="icon: arrow-left; ratio: 3" className="uk-position-large uk-position-top-left"></span>
            </a>
            <div className="uk-align-center uk-width-1-2@m">      
                <input placeholder="Name" className="uk-input uk-margin"></input>
                <Link to="/meetings/" className="uk-button uk-button-primary">Submit</Link>
            </div>
        </div>
    );
}
