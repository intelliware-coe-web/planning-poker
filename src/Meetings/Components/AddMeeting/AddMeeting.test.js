import AddMeeting from "./AddMeeting";

it('AddMeeting renders correctly', () => {
    expect(AddMeeting()).toMatchSnapshot(); 
});