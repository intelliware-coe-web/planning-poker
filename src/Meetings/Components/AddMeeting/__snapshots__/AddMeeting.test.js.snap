// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddMeeting renders correctly 1`] = `
<div
  className="uk-container uk-text-center@m uk-margin-top uk-margin-bottom"
>
  <h1
    className="uk-heading-small uk-heading-divider"
  >
    Add Meeting
  </h1>
  <a
    href="#/meetings"
  >
    <span
      className="uk-position-large uk-position-top-left"
      uk-icon="icon: arrow-left; ratio: 3"
    />
  </a>
  <div
    className="uk-align-center uk-width-1-2@m"
  >
    <input
      className="uk-input uk-margin"
      placeholder="Name"
    />
    <Link
      className="uk-button uk-button-primary"
      to="/meetings/"
    >
      Submit
    </Link>
  </div>
</div>
`;
